### DISCUSSION TITLE
Current SVN version - RMT Output broken

### DISCUSSION THREAD

**Discussion by jensrene** [forkineye/ESPixelStick#480] [General]
I have tried several things, but it seems the RMT WS2811 Output is broken and I don't understand it well enought to know how to fix.
I am using an ESP32 and in the first byte transmitted there is always one bit missing, making everything that follows wrong as well.

I got an analyzer dump showing the problem:

![RGBW](https://user-images.githubusercontent.com/100772742/156357055-62d7eb5a-6391-4840-be61-234486ab1426.PNG)

(2* RGBW LED with only RED at 255)

Does anyone else have this problem and maybe have an idea on how to fix it?

**Comment by MartinMueller2003** [comment]
I have a fix for this that should get into main by the end of April. There is a bug in the hardware that does not allow me to set a 1 tick output state so I had to split up the start bit into a 2 tick version.
